WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -201,7 +202,8 @@ + (NSDictionary *)dictionaryForElement:(id<FBXCElementSnapshot>)snapshot
   info[@"label"] = FBValueOrNull(wrappedSnapshot.wdLabel);
   info[@"rect"] = wrappedSnapshot.wdRect;
   
-  NSDictionary<NSString *, NSString * (^)(void)> *attributeBlocks = @{
+  // Define the base attribute blocks that apply to all elements.
+  NSDictionary<NSString *, NSString * (^)(void)> *baseAttributeBlocks = @{
```

Comment:
I think it makes sense to extract this logic into a separate method for better readability.

Also this dictionary should be defined as mutable to reduce the amount of boilerplate code